portfolio/src/main/webapp/style.css

```diff
@@ -268,9 +270,26 @@ a {
 }
 
 .gallery-card.wide {
+  background-position: left;
   grid-column: span 2;
 }
 
+.img-museum {
+  background-image:url(/images/IMG_1865.JPG)
```

Comment:
nit: add a space after `:`, here and below

Reply:
also, add a `;` at the end of the line, here and below

Reply:
Updated here 
bfcda3e